drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -1715,7 +1716,38 @@ static int vidioc_g_selection(struct file *file, void *priv,
 		}
 		break;
 	case ISP:
+		break;
 	case DEINTERLACE:
+		switch (s->target) {
```

Comment:
These need some extra validation. COMPOSE targets are only valid on V4L2_BUF_TYPE_VIDEO_CAPTURE, and CROP targets are only valid on V4L2_BUF_TYPE_VIDEO_OUTPUT